package.json

```diff
@@ -24,7 +24,6 @@
     "validator": "^10.11.0"
   },
   "devDependencies": {
-    "bootstrap": "^4.3.1",
-    "node-sass": "^4.14.1"
+    "bootstrap": "^4.3.1"
```

Comment:
The `bootstrap` dependency can be removed as well I think.

Reply:
Similarly, the `build:client` script in this file should be removed, and the section of the README file that talks about customizing the Bootstrap theme should also be removed.